e2e_test/e2e_bench_test.go

```diff
@@ -0,0 +1,72 @@
+package e2e_test
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/celo-org/celo-blockchain/core/types"
+	"github.com/celo-org/celo-blockchain/test"
+	"github.com/stretchr/testify/require"
```

Comment:
In other tests we started to use gomega matchers long time ago. To me, they seem quite helpful and complete. Wonder if it makes sense to use them, instead of adding another testing helper here

Reply:
I think it would be nice to standardize on a testing framework for use in the project. Both packages seem to be quite complete in terms of what they offer.

But if we wanted to do that I think we would have to go with testify since it has been in use in go-ethereum since 2018 (judging by the history of go.mod) and it seems unlikely that we would be able to easily replace all instances with gomega.

I also prefer testify because it seems more straightforward to use than gomega, and gomega really seems to be geared towards use with the ginko BDD framework, which AFAICS we are not using.